### PR TITLE
Add german translation

### PR DESCRIPTION
**Pull request by ErdbeerbaerLP**

### CODE REVIEW

File: locales/DE_de.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	"credits": {
+		"title": "Das Team",
+		"text": "Triff das Team, das hinter dem Projekt steckt",
+		"people": [
+			{
+				"name": "Jonathan Barrow (RedDucks)",
+				"caption": "Projektinhaber und leitender Entwickler",
+				"picture": "https://github.com/redduckss.png",
+				"github": "https://github.com/RedDuckss"
+			},
```
Comment (mrjvs) [inline locales/DE_de.json:77]: please change to new username:
```suggestion
			{
				"name": "Jonathan Barrow (jonbarrow)",
				"caption": "Projektinhaber und leitender Entwickler",
				"picture": "https://github.com/jonbarrow.png",
				"github": "https://github.com/jonbarrow"
			},
```

File: locales/GOOGLE_en.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
+{
```
Comment (mrjvs) [inline locales/GOOGLE_en.json:1]: while it is funny, please remove it from the pr, I cant merge it now 😄

**Review by mrjvs — CHANGES_REQUESTED** [review]

File: locales/GOOGLE_en.json (hunk 2 above)
Comment (ErdbeerbaerLP) [reply locales/GOOGLE_en.json]: Done

**Comment by montylion** [comment]
Looks good to me! Summoning @mrjvs just to make sure everything's ready to get merged

**Review by mrjvs — APPROVED** [review]

**Comment by mrjvs** [comment]
@montylion lgtm, can be merged

**Comment by montylion** [comment]
Great; @ErdbeerbaerLP, thanks for your contribution!

**Pull request merged**